UKHomeOffice/nmsw-ui
# NMSW-162 Help pages content

# Ticket

NMSW-162

----

## AC

As a data submitter
I want to be able to get help on how to use the system

----

## To test

- Run app
- > Landing page should match new content in the prototype linked in the ticket
- Go to sign in page
- > You should see a problems signing in section with a link to 'submit the required forms using email' (should match the prototype)
- Click on the link
- The link should take you to the help page (help page should match the prototype
- Scroll to the footer
- Click on the contact us link
- The page should match the prototype
- Click on the 'help section' link
- You should be taken to the help page (/help)
- Sign in
- Click on the help page in the nav
- You should be taken to the /help-page (logged in state of the page)
- Return to the contact us in the footer
- Click on the help section link
- > You should be taken to /help-page (logged in state) with the help nav link highlighted

----

## Notes
